input/europe/eu.csv

```diff
@@ -1,149 +0,0 @@
-Country,Name,Unit,Type,Model,Status,Net Capacity,Gross Capacity,Construction Date,Commerical Operation Date,Closure
```

Comment:
It has been removed. I thought having two files with similar data was confusing, so I left the PRIS data ( the more reliable one) and deleted the one from Wikidata